recipe/meta.yaml

```diff
@@ -48,18 +48,17 @@ test:
 
 about:
   home: https://github.com/mhammond/pywin32
-  # For more detail on the license look at the following issue
-  # https://github.com/mhammond/pywin32/issues/1127
   license: HPND AND BSD-3-Clause AND LGPL-2.1-or-later AND MIT
   license_file:
     - Pythonwin/License.txt  # BSD-3-CLause
     - Pythonwin/Scintilla/License.txt  # MIT
     - win32/License.txt  # BSD-3-CLause
     - com/License.txt  # BSD-3-CLause
-    - com/win32comext/mapi/src/mapi_stub_library/LICENSE  # MIT
```

Comment:
It looks like the license that was removed has just been relocated. 

`- com/win32comext/mapi/src/MAPIStubLibrary/LICENSE`